force_bdss/app/evaluate_operation.py

```diff
@@ -14,6 +14,13 @@ class EvaluateOperation(BaseOperation):
     """
 
     def run(self):
+
+        """verify the workflow.
+        Do not do use this [run(do_verify=False)] on unit tests :
+        use OptimizeOperation.run() to test verification.
+        """
+        self.verify_workflow()
+
         """ Evaluate the workflow. """
         mco_model = self.workflow.mco_model
         if mco_model is None:
```

Comment:
Lines 26-28 can be removed, since an error will be raised by `self.verify_workflow` beforehand if no `mco_model` is defined - this might also help the codecov